include/GafferScene/ScenePlug.h

```diff
@@ -160,19 +160,13 @@ class GAFFERSCENE_API ScenePlug : public Gaffer::ValuePlug
 
 			/// Standard constructors, for modifying context on the current thread.
 			PathScope( const Gaffer::Context *context );
-			[[deprecated("Use faster pointer version")]]
-			PathScope( const Gaffer::Context *context, const ScenePath &scenePath );
```

Comment:
I think we need to remove the implementation for all of these from the .cpp files too?